src/AppFrame.tsx

```diff
@@ -1,15 +1,28 @@
+/* eslint-disable @typescript-eslint/no-floating-promises */
 import React, { useCallback, useState } from "react";
 import { useLocation } from "react-router-dom";
 import { Frame, Layout, Navigation, Page, TopBar } from "@shopify/polaris";
 import { HomeMajor } from "@shopify/polaris-icons";
 import { Routes } from "./Routes";
 import { TwitterTimelineEmbed } from "react-twitter-embed";
+import { useTranslation } from "react-i18next";
+import i18next from "i18next";
 
 export function AppFrame() {
+  const { t, i18n } = useTranslation(undefined, { useSuspense: false });
   const location = useLocation();
   const [isMobileNavigationActive, setIsMobileNavigationActive] = useState(
     false,
   );
+  function languageToggle() {
+    // Display link to toggle to whichever language isn't set right now
+    // Get language code excluding country
+    // Initially language code set by browser so it is something like en-CA or en-US
+    if (i18n.language !== undefined && i18n.language.substring(0, 2) === "fr") {
```

Comment:
No action required, but do I remember hearing that we would support more than the two official languages? Maybe post-MVP? Should we create a ticket to update this toggle to a dropdown or something? cc @picuron

Reply:
I know the toggle only supports two languages, but we need someone to translate some resource strings first :)

Reply:
👍